Sources/SwiftBuildSupport/SwiftBuildSystem.swift

```diff
@@ -642,11 +642,20 @@ public final class SwiftBuildSystem: SPMBuildCore.BuildSystem {
 
         // Generate a table of any overriding build settings.
         var settings: [String: String] = [:]
-        // An error with determining the override should not be fatal here.
-        settings["CC"] = try? buildParameters.toolchain.getClangCompiler().pathString
-        // Always specify the path of the effective Swift compiler, which was determined in the same way as for the
-        // native build system.
-        settings["SWIFT_EXEC"] = buildParameters.toolchain.swiftCompilerPath.pathString
+
+        // If the SwiftPM toolchain corresponds to a toolchain registered with the lower level build system, add it to the toolchain stack.
+        // Otherwise, apply overrides for each component of the SwiftPM toolchain.
+        if let toolchainID = try await session.lookupToolchain(at: buildParameters.toolchain.toolchainDir.pathString) {
+            settings["TOOLCHAINS"] = "\(toolchainID.rawValue) $(inherited)"
+        } else {
+            // FIXME: This list of overrides is incomplete.
```

Comment:
issue: Do we have a GitHub issue tracking this `FIXME`?  If so, can we add it as a code comment?